### PR TITLE
Implement MODELDEL and SCRIPTDEL

### PR DESCRIPTION
**Pull request by lantiga**
Addresses #158 
/cc @K-Jo @MeirShpilraien

### CODE REVIEW

**Comment by K-Jo** [comment]
@lantiga needs docs

File: test/basic_tests.py [hunk 1]
```diff
@@ -99,6 +99,22 @@ def test_set_tensor(env):
...
+        model_pb = f.read()
+
+    con = env
+    ret = con.execute_command('AI.MODELSET', 'm', 'TF', 'CPU',
+                              'INPUTS', 'a', 'b', 'OUTPUTS', 'mul', model_pb)
+    con.assertEqual(ret, b'OK')
+
+    ret = con.execute_command('AI.MODELDEL', 'm')
+    con.assertEqual(ret, b'OK')
+
```
Comment (MeirShpilraien) [inline test/basic_tests.py]: I would have checked here if after the deletion the key does not exists

File: test/basic_tests.py [hunk 2]
```diff
@@ -580,6 +596,20 @@ def test_set_correct_script(env):
...
+    with open(script_filename, 'rb') as f:
+        script = f.read()
+
+    ret = env.execute_command('AI.SCRIPTSET', 'ket', 'CPU', script)
+    env.assertEqual(ret, b'OK')
+
+    ret = env.execute_command('AI.SCRIPTDEL', 'ket')
+    env.assertEqual(ret, b'OK')
+
```
Comment (MeirShpilraien) [inline test/basic_tests.py]: Same comment here

File: src/redisai.c [hunk 3]
```diff
@@ -1028,6 +1058,36 @@ int RedisAI_ScriptGet_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **argv
...
+
+  RedisModule_AutoMemory(ctx);
+
+  ArgsCursor ac;
+  ArgsCursor_InitRString(&ac, argv+1, argc-1);
+
+  RedisModuleString* keystr;
+  AC_GetRString(&ac, &keystr, 0);
+
+  RedisModuleKey *key = RedisModule_OpenKey(ctx, keystr, REDISMODULE_READ);
```
Comment (MeirShpilraien) [inline src/redisai.c]: Isn't this should be REDISMODULE_WRITE?

File: src/redisai.c [hunk 4]
```diff
@@ -580,6 +580,36 @@ int RedisAI_ModelGet_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **argv,
...
```
Comment (MeirShpilraien) [inline src/redisai.c]: REDISMODULE_WRITE?

File: src/redisai.c (hunk 3 above)
Comment (lantiga) [reply src/redisai.c]: Good catch, thanks!

**Comment by lantiga** [comment]
Thanks @K-Jo and @MeirShpilraien, I addressed both comments

**Pull request merged**